src/FilamentManager.php

```diff
@@ -64,4 +71,26 @@ public function userResource()
     {
         return config('filament.user_resource', UserResource::class);
     }
+
+    /**
+     * Configure Filament to not register its migrations.
+     *
+     * @return static
+     */
+    public static function ignoreMigrations()
+    {
+        static::$runsMigrations = false;
+
+        return new static;
+    }
+
+    /**
+     * Determine if Filament should run its migrations.
+     *
+     * @return bool
+     */
+    public static function runsMigrations()
```

Comment:
```suggestion
    public function shouldRunMigrations()
```

Reply:
@ryangjchandler I believe this should remain a static function so it can be called via the facade.